docs/usage/misc/metadata.md

```diff
@@ -13,13 +13,14 @@ Item metadata can be accessed through `Item.meta` using the hash syntax. The `It
 
 In addition to the Enumerable methods, the following methods are available for `Item.meta`:
 
-| Method     | Parameters               | Description                                | Example                          |
-| ---------- | ------------------------ | ------------------------------------------ | -------------------------------- |
-| clear      |                          | Deletes all metadata namespaces            | `Item1.meta.clear`               |
-| delete     | namespace                | Delete the given namespace                 | `Item1.meta.delete 'namespace1'` |
-| each       | namespace, value, config | Loops through all the item's namespaces    |                                  |
-| merge!     | **other                  | Merge a hash or other item's metadata      |                                  |
-| namespace? | namespace                | Returns true if the given namespace exists |                                  |
+| Method     | Parameters               | Description                                           | Example                          |
+| ---------- | ------------------------ | ----------------------------------------------------- | -------------------------------- |
+| clear      |                          | Deletes all metadata namespaces                       | `Item1.meta.clear`               |
+| delete     | namespace                | Delete the given namespace                            | `Item1.meta.delete 'namespace1'` |
+| each       | namespace, value, config | Loops through all the item's namespaces               |                                  |
+| merge!     | **other                  | Merge a hash or other item's metadata                 |                                  |
+| namespace? | namespace                | Returns true if the given namespace exists            |                                  |
+| dig        | key, *keys               | Dig through the namespace to find the specified value | `Item1.meta.dig('namespace1', 'foo', 'bar')                                 |
```

Comment:
Thanks! I'd like to see it in alphabetical order for easy scanning through the list.